independent-publisher-connectors/InstagramBasicDisplay/apiDefinition.swagger.json

```diff
@@ -266,6 +266,58 @@
           }
         ]
       }
+    },
+    "/refresh_access_token": {
+        "get": {
+            "responses": {
+                "default": {
+                    "description": "default",
+                    "schema": {
+                        "type": "object",
+                        "properties": {
+                            "access_token": {
+                                "type": "string",
+                                "description": "A long-lived Instagram User Access Token.",
+                                "title": "Access Token"
+                            },
+                            "token_type": {
+                                "type": "string",
+                                "description": "bearer",
+                                "title": "Token Type"
+                            },
+                            "expires_in": {
+                                "type": "integer",
+                                "description": "The number of seconds until the long-lived token expires.",
+                                "title": "Expires In",
+                                "format": "int32"
+                            }
+                        }
+                    }
+                }
+            },
+            "summary": "Refreshes unexpired long-lived Instagram User Access tokens.",
+            "description": "Long-lived tokens are valid for 60 days and can be refreshed as long as they are at least 24 hours old but have not expired, and the app user has granted your app the instagram_graph_user_profile permission. Refreshed tokens are valid for 60 days from the date at which they are refreshed. Tokens that have not been refreshed in 60 days will expire and can no longer be refreshed.",

```

Comment:
would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).